From 349814606e11c2f6fcbb0461ecebcfa96740725d Mon Sep 17 00:00:00 2001
Message-Id: <349814606e11c2f6fcbb0461ecebcfa96740725d.1349774431.git.minovotn@redhat.com>
In-Reply-To: <f076c6e7e12f7fd3ec2c6b171c4f154b28e2560b.1349774430.git.minovotn@redhat.com>
References: <f076c6e7e12f7fd3ec2c6b171c4f154b28e2560b.1349774430.git.minovotn@redhat.com>
From: Luiz Capitulino <lcapitulino@redhat.com>
Date: Fri, 5 Oct 2012 17:46:27 +0200
Subject: [PATCH 15/21] hmp.h: include qdict.h

RH-Author: Luiz Capitulino <lcapitulino@redhat.com>
Message-id: <1349459193-18226-14-git-send-email-lcapitulino@redhat.com>
Patchwork-id: 42761
O-Subject: [RHEL6.4 qemu-kvm PATCH 13/19] hmp.h: include qdict.h
Bugzilla: 832458
RH-Acked-by: Marcelo Tosatti <mtosatti@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

This is a manual backport of upstream commit
ff2f990b8212c8822d6a2f66ab4baeb48dae86bd. This is needed here because
hmp.h is relying on qapi-types.h including qdict.h. That's wrong,
as direct dependencies have to be explicitly included.

Signed-off-by: Luiz Capitulino <lcapitulino@redhat.com>
---
 hmp.h | 1 +
 1 file changed, 1 insertion(+)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 hmp.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hmp.h b/hmp.h
index 9751498..7e87148 100644
--- a/hmp.h
+++ b/hmp.h
@@ -15,6 +15,7 @@
 #define HMP_H
 
 #include "qemu-common.h"
+#include "qdict.h"
 
 #ifdef CONFIG_LIVE_SNAPSHOTS
 #include "rhev-qapi-types.h"
-- 
1.7.11.4

